database/postgres/dal/recommendation.py

```diff
@@ -0,0 +1,110 @@
+from database.models import Likes, Views, News, User, UserClubPreferences, \
+    UserPreference, Sport, TeamInNews, UserRecommendations
+from sqlalchemy import union_all, literal, func
+
+
+class RecommendationDAL:
+    def __init__(self, session=None):
+        self.session = session
+
+
+    def get_all_users(self):
```

Comment:
Seems like this one belongs to UserDAL and has nothing to do with Recs.

I'm also concerned with ".all()"